pkgs/top-level/all-packages.nix

```diff
@@ -9416,7 +9416,7 @@ with pkgs;
   naabu = callPackage ../tools/security/naabu { };
 
   nagstamon = callPackage ../tools/misc/nagstamon {
-    pythonPackages = python3Packages;
+    qt6 = qt6Packages;
```

Comment:
Is there a reason for not directly using `qt6Packages` in the derivation inputs ?